testsuite/tests/workflows/air-gapping/test.py

```diff
@@ -0,0 +1,90 @@
+"""
+Test fetching a crate online, and subsequently building offline.
+"""
+
+
+import os
+import shutil
+import subprocess
+import sys
+
+from drivers.alr import run_alr
+from drivers.helpers import init_git_repo, on_windows
+from drivers.asserts import assert_eq, assert_match
+
+
+# Mock git, curl, gprbuild etc. with dummy scripts
+if on_windows():
+    print('SKIP: command mocking unavailable on Windows')
+    sys.exit(0)
```

Comment:
Rather than hardcoding this detection in the test, look at the test skels examples for using the `control` field in the yaml file.

Reply:
I hadn't noticed `skip_unix` was an option; thanks for pointing that out.